src/components/sheetSettings.jsx

```diff
@@ -36,6 +37,7 @@ class Settings extends Component {
                         min={300}
                         max={2000}
                         defaultValue={this.state.settings.width || 816}
+                        onChange={this.handleSettingsChange}
```

Comment:
This strikes me as a bad idea, every click on the number input will reload the page, which will be about 100 reloads of state in 5 seconds!